TESTNET_DEPLOY.md

```diff
@@ -0,0 +1,17 @@
+# Testnet deploy steps
+
+1. Checkout to the commit/branch that should be deployed with `git checkout <ref>`.
+2. Do some last code checks.
+    - Make sure all tests pass by running `cargo test`
+    - Make sure the runtime migrations work by running `just try-runtime-upgrade-testnet`  
+3. Create the release tag following the format `runtime/{test,main}net-{spec_version}`. Example: runtime/testnet-12 (the spec version can be found in `runtime/src/lib.rs` under the `runtime_macro!` call). This will trigger a Github Action that will build the runtime wasm binary.
+4. Download said runtime binary and check it's checksum.
```

Comment:
Add the link to where the binary can be found and instruct to download the compact and compressed version. As well as how to test for the hash checksum (command and such)